packages/php/composer.json

```diff
@@ -24,7 +24,6 @@
   },
   "require-dev": {
     "phpunit/phpunit": "^9.5",
-    "mockery/mockery": "^1.4",
```

Comment:
While getting the PHP integration suite working I uncovered a handful of problems with the way I was mocking out Laravel's `Request` class that I decided it was easier to fill a real instance of it with fake data rather than creating a fake partial instance with fake data.